src/main/java/com/google/api/codegen/transformer/nodejs/NodeJSGapicSurfaceTransformer.java

```diff
@@ -265,16 +265,29 @@ private ViewModel generateApiClass(GapicInterfaceContext context) {
     String version = namer.getApiWrapperModuleVersion();
     boolean hasVersion = version != null && !version.isEmpty();
     ArrayList<VersionIndexRequireView> requireViews = new ArrayList<>();
+    SurfaceNamer surfaceNamer =
+        new NodeJSSurfaceNamer(productConfig.getPackageName(), NodeJSUtils.isGcloud(productConfig));
+    FeatureConfig featureConfig = new NodeJSFeatureConfig();
     for (Interface apiInterface : apiInterfaces) {
       Name serviceName = namer.getReducedServiceName(apiInterface);
       String localName =
           hasVersion ? serviceName.join(version).toLowerCamel() : serviceName.toLowerCamel();
+      ModelTypeTable modelTypeTable =
+          new ModelTypeTable(
+              new JSTypeTable(productConfig.getPackageName()),
+              new NodeJSModelTypeNameConverter(productConfig.getPackageName()));
+      GapicInterfaceContext context =
+          GapicInterfaceContext.create(
```

Comment:
I think context creation can be moved to a separate method to avoid code duplication (there are two places to create context in this class).

Refer to https://github.com/googleapis/toolkit/blob/master/src/main/java/com/google/api/codegen/transformer/ruby/RubyGapicSurfaceTestTransformer.java#L248